### DISCUSSION TITLE
hiSense LTE3415-SH+ // 2.5 Gb negotiation

### DISCUSSION THREAD

**Discussion by gianry** [Anime4000/RTL960x#293] [Q&A]
Hi,
I'm using the hiSense LTE3415-SH+ gpon in a 2.5 Gb enabled media converter. I'm not able to negotiate an ethernet link at 2.5 gigabit. Could you help me to understand if it is necessary a firmware mode or just change LAN_SDS_mode parameter ? 
thanks

**Comment by Anime4000** [comment]
what Media Converter are you using?

**Reply by gianry** [reply]
HI,  a chinese 2.5 / 1 gigabit 
I've the stock firmware on the Gpon stick (in italy it has been sold by Technicolor )
